### ISSUE TITLE
export pdf or svg vector file

### ISSUE THREAD

**Issue by jtkeyva** [promops/flutter_pretty_qr#42]
thanks for the nice package! any plans to allow exporting of vector formats?

**Comment by promops** [comment]
@jtkeyva Hello! Thanks for the idea, I’ll think about how to realize this well

**Comment by jtkeyva** [comment]
yeah it would be great for high res print and svg could scale in apps nicely
